ServerCore/wwwroot/lib/jquery-validation-1.19.2-dist/jquery.validate.js

```diff
@@ -1426,11 +1426,15 @@ $.extend( $.validator, {
 		// https://jqueryvalidation.org/url-method/
 		url: function( value, element ) {
 
+			/* Included:
```

Comment:
??? Why is this in the PR?

Reply:
See last bullet point in PR description.  I'm surprised you never hit this issue with your local development!

Reply:
I haven't run into this. Can we add this to the top of Puzzles/Edit/OnPostAsync instead? I'm just terrified of changing libraries that we may choose to update in the future.

```
            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == Environments.Development)
            {
                ModelState.ClearValidationState("Puzzle.CustomURL");
                ModelState.ClearValidationState("Puzzle.CustomSolutionURL");
            }
```